dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/AbstractNodeVisitor.java

```diff
@@ -380,4 +380,10 @@ public R visitParameterizedType(ParameterizedTypeNode n, P p) {
     public R visitMarker(MarkerNode n, P p) {
         return visitNode(n, p);
     }
+
+    // Merge store after an expression statement
```

Comment:
```suggestion
```

Reply:
I would remove this comment - it's not what this method does and might not be what an actual subclass does.